Resources/Prototypes/Entities/Objects/Weapons/Melee/spear.yml

```diff
@@ -29,3 +25,12 @@
   length: 0.10
   speed: 6
   arcType: Poke
+
+- type: entity
+  name: poisonable spear
+  parent: Spear
+  id: PoisonableSpear
+  components:
+  - type: MeleeChemicalInjector
+  - type: SolutionContainer
+    maxVol: 5
```

Comment:
IMO it should just be on the regular weapons rather than its own prototype so someone can splash chems onto a spear for example (might need to add an IInteractUsing to the component? Not sure).

Reply:
Put it on the base spear, we don't have much else that makes thematic sense to be poisonable yet.